script.js

```diff
@@ -140,7 +154,9 @@ function update () {
     }
 
     processNextMove(player, PLAYER_SPEED);
-    
+    for (let i = 0; i < ghosts.children.entries.length; i++) {
+        processNextMove(ghosts.children.entries[i], GHOST_SPEED, true);
+    }
```

Comment:
We could use [Array.forEach()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) here.